### PR TITLE
Updating people and projects for OLS-2

### PR DESCRIPTION
**Pull request by malvikasharan**
# Summary

This PR should inlcude the following

- [x] Add OLS-2 mentees
- [x] Add OLS-2 projects
- [x] Add countries for all people
- [x] Blogpost announcing the OLS-2 participants

### CODE REVIEW

**Comment by bebatut** [comment]
I created the script to add people information to `people.yaml` from a sheet (+ documentation) and added OLS-2 mentees.

I also drafted the post for OLS-2 announcement.

File: _posts/2020-09-01-ols2-announcement.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+{% assign p-mentors = all-mentors | remove_first: ', ' | split: ", " | uniq | sort %}
+{% assign m-countries = all-m-countries | remove_first: ', ' | split: ", " | uniq | sort %}
+
+We are thrilled to announce that [{{ p-participants | size }} members](/{{ cohort }}/projects-participants/#participants), who are the project leads of [{{ projects | size }} diverse projects](/{{ cohort }}/projects-participants/#projects), have joined the second cohort of the Open Life Science mentoring program - OLS-2!
+
+### Meet our mentees!
+
+The mentees joining this program are {{ p-participants | join: ', ' }}. These individuals are based in {{ p-countries | size }} countries ({{ p-countries | join: ', ' }}) where they will be leading their respective projects. 
+
+Their projects range from TODO.
```
Comment (bebatut) [inline _posts/2020-09-01-ols2-announcement.md:81]: Need to be filled

File: ols-2.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ OLS's second cohort (OLS-2) will be conducted from September 2020 until December
 
 Participants join this program with a project that they either are already working on or want to develop during this program. More details about the role of a project lead (mentee) can be found [here](/about#mentees).
 
-<!--For the second round of the Open Life Science program, we are happy to have [{{ p-participants | size }} participants]-->
+For the second round of the Open Life Science program, we are happy to have [{{ p-participants | size }} participants](/ols-2/projects-participants#participants) with [{{ projects | size }} projects](/ols-2/projects-participants#projects).
```
Comment (malvikasharan) [inline ols-2.md:107]:
```suggestion
For the second round of the Open Life Science program, we welcome [{{ p-participants | size }} participants](/ols-2/projects-participants#participants) with [{{ projects | size }} projects](/ols-2/projects-participants#projects). 
```

File: _posts/2020-09-01-ols2-announcement.md (hunk 1 above)
Comment (malvikasharan) [inline _posts/2020-09-01-ols2-announcement.md:81]:
```suggestion
Topics for their projects include data-centric engineering, ethical research, data communication, developing community aspects of a python package, enhancing lab automation, creating mentorship network, collaborative training delivery, open science education, online database for 3D modeling workflows, establishing disease-related preprints, applying machine learning practices,  interdisciplinary approaches to infections and diseases, curating land-use land-cover data, science campaigning, metagenomic pipeline, Galaxy, applying the FAIR principle in a new field, creating a reproducible environment and open hardware, enhancing accessibility in community and citizen science projects, and fundraising and ensuring project sustainability.
```

File: _posts/2020-09-01-ols2-announcement.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+- malvikasharan
+- yochannah
+image: https://images.unsplash.com/photo-1489533119213-66a5cd877091?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1051&q=80
+photos:
+  name: Danielle MacInnes
+  license: CC-BY
+  url: https://unsplash.com/photos/IuLgi9PWETU
+---
+
+*We are excited to welcome this fantastic set of mentees who are not only located in different parts of the world, but also represent different identities and backgrounds, aim to address a wide range of questions in their field, and are motivated to bring a culture change in their areas. Many of them are long-standing Open Scientists who aim to use this opportunity to apply “Open by Design” principle in their projects through this program.*
```
Comment (malvikasharan) [inline _posts/2020-09-01-ols2-announcement.md:15]:
```suggestion
*We are excited to kick-off the second cohort of Open Life Science with another incredible cohort of mentors, mentees, and experts. We are honored to bring together members of diverse identities and backgrounds who represent expertise from different domains of research, who are working to address a wide range of relevant questions in their field, and are motivated to bring a culture change in their areas. Many of them are long-standing Open Scientists who aim to use this opportunity to apply open research and community-based principle in their projects through this program.*
```

File: _posts/2020-09-01-ols2-announcement.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+### Meet our mentees!
+
+The mentees joining this program are {{ p-participants | join: ', ' }}. These individuals are based in {{ p-countries | size }} countries ({{ p-countries | join: ', ' }}) where they will be leading their respective projects. 
+
+Their projects range from TODO.
+
+### Meet our mentors!
+
+Our mentees will be supported in this program by our mentors' community who have been paired based on the compatibility of expertise and interests of mentors with the requests and requirements of our mentees. Our mentors are Open Science champions with previous experiences in training and mentoring. They are currently working in different professions in data science, publishing, community building, software development, clinical studies, industries, scientific training and IT services.
```
Comment (malvikasharan) [inline _posts/2020-09-01-ols2-announcement.md:85]:
```suggestion
Our project leads (aka mentees) have been paired with 1 or 2 mentors based on their specific requirements of expertise and interests along with time zones and language preferences. Our mentors are Open Science champions with previous experiences in training, mentoring, computing, and community skills. They are currently working in different professions in data science, education, citizen science, publishing, community building, software development, clinical studies, industries, scientific training, policymaking, IT services, and so on. 

Additionally, we have an incredible experts' community who will be delivering specialised talks during the cohort calls and will be available for our project leads for expert consultations upon request.
```

File: _posts/2020-09-01-ols2-announcement.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+We welcome our {{ p-mentors | size }} mentors, {{ p-mentors | join: ', ' }}, based in {{ m-countries | size }} countries ({{ m-countries | join: ', ' }}). {{ prev-part-count }} of them were participants and {{ prev-mentor-count }} mentors in the [previous cohort (OLS-1)](/ols-1). They will be supported by [{{ experts | size }} experts](/{{ cohort }}#experts).
+
+We are extremely grateful to them for their support and contributions to OLS and their impactful work in other open communities. They are committed to supporting their mentees in this program to help create a more open and fair-research, knowledge-sharing and inclusive culture within life science.
```
Comment (malvikasharan) [inline _posts/2020-09-01-ols2-announcement.md:89]:
```suggestion
We are extremely grateful to them for their support and contributions to OLS and their impactful work in other open communities. They are committed to supporting their mentees in this program to help create a more open and fair-research, knowledge-sharing and inclusive culture within life science and beyond.
```

File: _posts/2020-09-01-ols2-announcement.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+### The Program
+
+We will begin our program this week with a mentoring training call, and mentor-mentee introductions. Check out the complete schedule and plans for OLS-2 here: [{{ site.url }}/{{ cohort }}](/{{ cohort }}).
```
Comment (malvikasharan) [inline _posts/2020-09-01-ols2-announcement.md:93]:
```suggestion
We begin our program this week with a mentoring training call and mentor-mentee introductions. Check out the complete schedule and plans for OLS-2 here: [{{ site.url }}/{{ cohort }}](/{{ cohort }}).
```

File: _posts/2020-09-01-ols2-announcement.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+You can keep track of our program, the progress of our second cohort and future announcements by following our twitter profile [@{{ site.twitter }}](https://twitter.com/{{ site.twitter }}) or subscribe to [our announcements list](site.announcement_list).
+
+We invite new contributions to the program as a [new issue on the GitHub repo]({{ site.github.repository_url }}/issues) or by [email to the team](mailto:{{ site.email }}).
+
+Once again, let's welcome our mentors and mentees to this program!
```
Comment (malvikasharan) [inline _posts/2020-09-01-ols2-announcement.md:99]:
```suggestion
Once again, let's welcome our mentors, mentees and experts to this program!
```

**Comment by malvikasharan** [comment]
@bebatut All of these look great and as usual a lot of work. 💪🙌 Thank you so much for your work. I have added some suggestions (mostly in the blogpost).

**Comment by bebatut** [comment]
Thanks @malvikasharan for the suggestions. It should be ready now. @yochannah do you want to check?

File: _posts/2020-09-01-ols2-announcement.md (hunk 1 above)
Comment (malvikasharan) [reply _posts/2020-09-01-ols2-announcement.md]: added in my review below

**Comment by yochannah** [comment]
tested locally on my machine - looks fab. Great job both!

**Pull request merged**